### ISSUE TITLE
Normal Mode: can't use `g` more then once. Always jumps to zoom title (ie as if `gg` was invoked)

### ISSUE THREAD

**Issue by MaxLinWorm** [Wojnach/vimflowy#33]
First off, AMAZING extensions.

I don't know if it is by design, but I can't traverse up a hierarchy to an item's parent more than once. If I press two `g` in a row, even after some time, the cursor always focuses the zoom parent (the root of the current zoom level). I'd think the `gg` keybinding would have a short time out no?

**Comment by Wojnach** [comment]
yes its by design atm. But I agree with you, it would be nice to be able to traverse up the hierarchy, to an item's parent, everytime you press G, in rapid succession. Probably more useful than GG tbh. I'd maybe vote for removing GG in favour of being able to traverse up the hierarchy fast instead. A timer would work as well but I imagine it might be annoying if you don't get the timing down correctly and accidentaly trigger GG.

**Comment by Wojnach** [comment]
should be fixed in the latest commit. I've disabled g,g in favor of just having g. offical chrome extensio will be updated during the week
